ci/build.yml

```diff
@@ -250,33 +250,6 @@ jobs:
         trigger_sha: '$(trigger_sha)'
     - template: report-end.yml
 
-- job: build_canton_3x_with_bazel
-  dependsOn:
-    - check_for_release
-    - build_canton
-  variables:
-    - name: release_sha
-      value: $[ dependencies.check_for_release.outputs['out.release_sha'] ]
-    - template: job-variables.yml
-  timeoutInMinutes: 60
-  pool:
-    name: 'ubuntu_20_04'
-    demands: assignment -equals default
-  steps:
-    - template: report-start.yml
-    - checkout: self
-    - bash: |
-        set -euo pipefail
-        git checkout $(release_sha)
-        if [ -x ci/build-canton-3x.sh ]; then
```

Comment:
This file has never existed on this branch AFAICT so this job never did much of anything.